common/src/main/java/com/codeforcommunity/email/EmailOperations.java

```diff
@@ -150,6 +151,34 @@ private void sendEmail(Email email, String subject) {
     }
   }
 
+  private Email buildEmailSingleRecipient(
+          String sendToName, String sendToEmail, String subject, String emailBody, List<AttachmentResource> attachments) {
+    Email email =
+            EmailBuilder.startingBlank()
+                    .from(senderName, sendEmail)
+                    .to(sendToName, sendToEmail)
+                    .withSubject(subject)
+                    .withHTMLText(emailBody)
+                    .withAttachments(attachments)
+                    .buildEmail();
+
+    return email;
+  }
+
+  /**
+   * Send an email with the given subject and body to the user with the given name at the given
+   * email with attachments.
+   */
+  public void sendEmailToOneRecipient(
+          String sendToName, String sendToEmail, String subject, String emailBody, List<AttachmentResource> attachments) {
+    if (!shouldSendEmails) {
+      return;
+    }
+    logger.info(String.format("Sending email with subject `%s`", subject));
+    Email email = buildEmailSingleRecipient(sendToName, sendToEmail, subject, emailBody, attachments);
+    this.sendEmail(email, subject);
+  }
```

Comment:
⛏️ The helper method `buildEmailSingleRecipient` is a good idea, but is there any reason we can't use regular Java method overloading here and have the other version of `sendEmailToOneRecipient` just call this version and pass an empty list for `attachments`? IMO that would keep things simple and avoid code duplication.

Reply:
I was considering but was afraid to break the existing non-attachment email functionality. Will refactor